docs/04-operation-guides/troubleshooting/apix-02-401-unauthorized-403-forbidden.md

```diff
@@ -0,0 +1,92 @@
+---
+title: Cannot connect to a service exposed by an API Rule - 401 Unauthorized or 403 Forbidden
+---
+
+## Symptom
+
+You reach your service and get `401 Unauthorized` or `403 Forbidden` in response.
+
+## Remedy
+
+In such case, make sure that:
+
+- You are using an access token with proper scopes and it is active:
+
+  1. Export the credentials of your OAuth2Client as environment variables:
+
+      ```bash
+      export CLIENT_ID="$(kubectl get secret -n $CLIENT_NAMESPACE $CLIENT_NAME -o jsonpath='{.data.client_id}' | base64 --decode)"
+      export CLIENT_SECRET="$(kubectl get secret -n $CLIENT_NAMESPACE $CLIENT_NAME -o jsonpath='{.data.client_secret}' | base64 --decode)"
+      ```
+
+  2. Encode your client credentials and export them as an environment variable:
+
+      ```bash
+      export ENCODED_CREDENTIALS=$(echo -n "$CLIENT_ID:$CLIENT_SECRET" | base64)
+      ```
+
+  3. Check access token status:
+
+      ```bash
+      curl -X POST "https://oauth2.{DOMAIN}/oauth2/introspect" -H "Authorization: Basic $ENCODED_CREDENTIALS" -F "token={ACCESS_TOKEN}"
+      ```
+
+  4. Generate a [new access token](../../03-tutorials/00-api-exposure/apix-03-expose-and-secure-service.md#register-an-oauth2-client-and-get-tokens) if needed.
+
+- Your client from OAuth2Client resource is registered properly in Hydra OAuth2 and OpenID Connect server. You need to call the Hydra administrative endpoint `/client` from inside of the cluster. Follow this steps:
+
+  1. Fetch the Client ID from Secret specified in the OAuth2Client resource:
+
+      ```bash
+      kubectl get secrets {SECRET_NAME} -n {SECRET_NAMESPACE} -o jsonpath='{ .data.client_id }' | base64 --decode
+      ```
+
+  2. Create a simple curl Pod:
+
+      ```yaml
+      ---
+      apiVersion: v1
+      kind: Pod
+      metadata:
+        labels:
+          app: ory-curl
+        name: ory-curl
+        namespace: {SECRET_NAMESPACE}
+      spec:
+        containers:
+        - name: curl
+          image: alpine
+          terminationMessagePolicy: "FallbackToLogsOnError"
+          command:
+            - /bin/sh
+            - -c
+            - |
+              apk add curl jq
+              curl ory-hydra-admin.kyma-system.svc.cluster.local:4445/clients | jq '.'
+      ```
+
+  3. Check logs from the `ory-curl` Pod:
+
+      ```bash
+      kubectl logs -n {SECRET_NAMESPACE} ory-curl curl
+      ```
+
+  4. If the Client ID from step 1 is not available on the clients list, make sure Hydra has access to the database and/or restart the Hydra Measter Pod.
+  You can check the logs using the following commands:
```

Comment:
Be aware that these are rendered as one line (and the numbering uses Roman numbers, as it's under an unordered list item). 
<img width="745" alt="image" src="https://user-images.githubusercontent.com/53601578/153406029-e76ed265-a723-481a-9359-b3d93f837982.png">

Reply:
I fixed the formatting so that the commands are in two separate code snippets.

Reply:
Oh? I still see it [displayed as one line](https://github.com/kyma-project/kyma/blob/b8d0bd064e60103925f98c4cdb8b49737d9820bc/docs/04-operation-guides/troubleshooting/apix-02-401-unauthorized-403-forbidden.md). Not that I mind, it's perfectly fine, I just noticed that you put the second sentence in a new line, so wanted to make sure you're aware of it. 
![image](https://user-images.githubusercontent.com/53601578/154330218-db52bf28-40d4-4798-9a08-825a737fd217.png).

I have an impression we're talking about two different things 🤔